Sources/MapboxNavigation/NavigationMapView+VanishingRouteLine.swift

```diff
@@ -47,12 +47,14 @@ extension NavigationMapView {
     }
     
     /**
-     Update the existing primary route line during active navigation.
+     Update the existing primary route line during active navigation when refresh or reroute event occurs.
      
      - parameter routeProgress: The current `RouteProgress`.
      - parameter coordinate: The current user location coordinate.
      */
-    func updateRouteLine(routeProgress: RouteProgress, coordinate: CLLocationCoordinate2D?) {
+    public func updateRouteLine(routeProgress: RouteProgress, coordinate: CLLocationCoordinate2D?) {
```

Comment:
I would suggest include some suffix to the method name so that it explicitly states that route line will be redrawn from the 0 fraction traveled, or alternatively add argument flag to control resetting `fractionTraveled`

Reply:
Thanks for the suggestion, I'll update the method name.

But the issue is that the route line will be redraw because of the refresh and rerouting event. But after the method, the `fractionTraveled` will not be `0.0`, it's to make sure the `currentLineGradientStops` and `currentRestrictedAreasStops` could be generated from the start with the new route line.